entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+# Docker entrypoint
+
+# Source: https://github.com/jschnasse/read-env-example
+function generateConfigJs() {
+    echo "/*<![CDATA[*/";
+    echo "window.env = window.env || {};";
+    for i in `env | grep '^REACT_APP_'`
```

Comment:
prefer `$(cmd)`over `` `cmd` ``

Reply:
Also, I don't think its mentioned anywhere that the environment variables need to start with `REACT_APP_`